mitodl/mitxpro
# Add staff payment_type to CouponPaymentVersion

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #893 

#### What's this PR do?
Adds "staff" as a valid choice for `CouponPaymentVersion.payment_type` and as a choice in the coupon creation form.

#### How should this be manually tested?
Go to `http://xpro.odl.local:8053/ecommerce/admin/coupons` and create a single-use coupon.  Choose a payment type of 'staff' and enter whatever you want for transaction number.  Submit the form, the coupons should be successfully created.
